modules/psap-special-resource-operator-installing-using-web-console.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-special-resource-operator.adoc
+
+[id="installing-the-special-resource-operator-using-web-console_{context}"]
+= Installing the Special Resource Operator by using the web console
+
+As a cluster administrator, you can install the Special Resource Operator (SRO) by using the {product-title} web console.
+
+[NOTE]
+====
+It is recommended that you create the `Namespace` as detailed in the previous section.
```

Comment:
We don't normally like to mention the "previous section", because it's possible that these modules could get separated, or reordered.  Instead, I'd just add a step to the below procedure that shows how to create the namespace. You can do it through the web console as well.

I believe it's through Administration -> Namespaces, but you can test on a cluster, or can probably find the right steps in another module in our repo.

Reply:
@dagrayvid - Would this work?
. Log in to the {product-title} web console.
. Create the required namespace for the Special Resource Operator.
.. Navigate to *Administration* -> *Namespaces* and click *Create Namespace*.
.. Enter `openshift-special-resource-operator` in the *Name* field and click *Create*.

Reply:
Yes, looks good.